Test_Receiver/BMS_TestReceiver.cpp

```diff
@@ -0,0 +1,12 @@
+#define CATCH_CONFIG_MAIN  // This tells Catch to provide a main() - only do this in one cpp file
+#include "../Test_Sender/catch.hpp"
+#include "../BMS_Receiver/dataBmsRx.h"
+
```

Comment:
These tests miss the content of the output, which is the core of sender functionality. A maintainer of this code can inadvertently make a bug which changes the output format. This will be caught only when we integrate with the receiver, which will then take a  lot more time to diagnose and solve.